extensions/SiteMapIndex/lib/Util.pm

```diff
@@ -145,11 +123,15 @@ sub _generate_sitemap_index {
 <sitemapindex xmlns="http://www.sitemaps.org/schemas/sitemap/0.9">
 END
 
+  my $sitemap_url
+    = 'https://'
+    . Bugzilla->params->{sitemapindex_s3_bucket}
+    . '.s3-us-west-2.amazonaws.com';
```

Comment:
Is `us-west-2` always guaranteed? Maybe that should be configurable in the same way as the bucket name.

Reply:
dlactin states that it will unlikely change unless we need to provision resources in another region due to a major outage. I will merge this and open a new bug for adding a config option for region.